src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEAssemblySymbol.cs

```diff
@@ -280,5 +283,17 @@ internal PEModuleSymbol PrimaryModule
         }
 
         public override AssemblyMetadata GetMetadata() => _assembly.GetNonDisposableMetadata();
+
+        internal bool GetCompilerFeatureRequiredUseSiteInfo(ref UseSiteInfo<AssemblySymbol> result, MetadataDecoder decoder)
```

Comment:
>MetadataDecoder decoder [](http://example.com/codeflow?start=100&length=23)

Do we really have to pass the decoder here? Can't we create it one for the primary module demand?